docs/get-started/running-nodes.mdx

```diff
@@ -4,306 +4,218 @@ title: 'Running Etherlink Nodes'
+   For more information about building Octez binaries, see the Octez documentation: https://tezos.gitlab.io/index.html
+
+   - You can also use Octez binary builds.
+   Builds for Linux systems are available for the [x86_64](https://gitlab.com/tezos/tezos/-/jobs/6849203975/artifacts/browse/octez-binaries/x86_64/) architecture.
+
+1. Initialize the local context of the node, which is where it stores local data:
+
+   1. Set the environment variable `sr_observer_data_dir` to the directory where the node should store its local data.
+   1. Initialize the local context by running this command:
+
+      ```bash
+      octez-smart-rollup-node init observer config for sr1Ghq66tYK9y3r8CC1Tf8i8m5nxh8nTvZEf  \
+        with operators --data-dir $sr_observer_data_dir
+      ```
+
+      This command generates a minimal configuration file (`config.json`) in the local data folder:
+
+      ```json
+      { "smart-rollup-address": "sr1Ghq66tYK9y3r8CC1Tf8i8m5nxh8nTvZEf",
+        "smart-rollup-node-operator": {}, "fee-parameters": {},
+        "mode": "observer" }
+      ```
+
+1. Activate the `unsafe-pvm-patches` setting, which is required for Etherlink, and add the location of the preimages by adding these settings to the `config.json` file, as in this example:
+
+   ```json
+   { "smart-rollup-address": "sr1Ghq66tYK9y3r8CC1Tf8i8m5nxh8nTvZEf",
+     "smart-rollup-node-operator": {}, "fee-parameters": {},
+     "unsafe-pvm-patches": [
+       { "increase_max_nb_tick": "50_000_000_000_000" }
+     ],
+     "pre-images-endpoint": "https://snapshots.eu.tzinit.org/etherlink-mainnet/wasm_2_0_0",
+     "mode": "observer" }
+   ```
+
+   Similar to the Rust keyword, `unsafe` here means to be used with a good understanding of the feature.
+   Etherlink uses this setting with care and using it is safe in this context.
+
+1. Start the Smart Rollup node in observer mode by running this command and using the RPC endpoint of a layer 1 node that is running in archive mode:
+
+   ```bash
+   octez-smart-rollup-node --endpoint https://rpc.tzkt.io/mainnet run \
+     --data-dir $sr_observer_data_dir \
+     --apply-unsafe-patches
+   ```
+
+   As in this example, you can use a public layer 1 RPC node for testing purposes.
+   For production use, you must use a layer 1 node that is running in archive mode and provides access to its RPC endpoints.
+
+   The `--apply-unsafe-patches` setting is required only the first time that you start the node.
+   After that you can omit it.
+   After the first run, you can also remove the `unsafe-pvm-patches` field from the configuration file.
+
+   After the Smart Rollup node has caught up with the current state via the archive node, you can safely connect it to a rolling layer 1 node.
+
+1. Verify that the Smart Rollup node is running by querying it.
+For example, this query gets the latest block:
+
+   ```bash
+   curl -s http://localhost:8932/global/block/head
```

Comment:
I think there is an RPC for checking the health of a rollup node, I'll check

Reply:
there is a `/health` or a `/ping` RPC

Reply:
Changed to /health.